ui.frontend/src/view/FormFieldBase.js

```diff
@@ -598,16 +604,45 @@ class FormFieldBase extends FormField {
 
     /**
      * Updates the HTML state based on the description state of the field.
-     * @param {string} description - The description.
-     */
-    updateDescription(description) {
-        if (this.description) {
-            this.description.querySelector("p").innerHTML = description;
-        } else {
-            //TODO: handle the case when description is not present initially.
+     * @param {string} descriptionText - The description.
+     */
+    updateDescription(descriptionText) {
+        if (descriptionText !== undefined) {
```

Comment:
```  
  if (typeof descriptionText !== 'undefined') {
        const descriptionElement = this.getDescription();
        if (descriptionElement) {
            let pElement = descriptionElement.querySelector("p");
            if (!pElement) {
                // If the description is updated via rule then it might not have <p> tags
                pElement = document.createElement('p');
                descriptionElement.appendChild(pElement);
            }
            pElement.textContent = descriptionText;
        } else {
            this.#addDescriptionInRuntime(descriptionText);
        }
    }
    ```